eos-copy-host-mali-driver

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -ex
+#
+# eos-copy-host-mali-driver: Copy the host's GL drivers into a flatpak local
+# extension directory.
+#
+# Copyright (C) 2017 Endless Mobile, Inc.
+# Authors:
+#  Philip Chimento <philip@endlessm.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+EXTENSIONS_DIR=/var/lib/flatpak/extension
+if test -d /var/endless-extra/flatpak; then
+    EXTENSIONS_DIR=/var/endless-extra/flatpak/extension
+fi
+
+DEST_DIR="$EXTENSIONS_DIR/org.freedesktop.Platform.GL.host/arm/1.6/lib"
+mkdir -p "$DEST_DIR"
```

Comment:
I thought that's what it should be; the driver is in a `mali400-blahblah` subdir on the host system as well. But maybe flatpak is not looking for it there? I'll change it.